src/emucore/Random.hxx

```diff
@@ -55,8 +56,8 @@ class Random
     uInt32 next();
 
   private:
-    // Indicates the next random number
-    uInt32 myValue;
+    // Mersenne Twister object to randomization
+    static std::mt19937 rndGenerator;
```

Comment:
This is a C++11 feature :( It might be nice to have a fallback in the event that someone compiles the ALE with a non-C++11 compiler (they're still around right now, unfortunately).